content/docs/components/misc/metadata.md

```diff
@@ -4,43 +4,183 @@ description = "Tracking and managing metadata of machine learning workflows in K
 weight = 5
 +++
 
-The goal of the [Metadata](https://github.com/kubeflow/metadata) project is to help Kubeflow users understand and manage their machine learning workflows by tracking and managing the metadata of workflows.
+The goal of the [Metadata](https://github.com/kubeflow/metadata) project is to 
+help Kubeflow users understand and manage their machine learning (ML) workflows
+by tracking and managing the metadata that the workflows produce. 
 
-## Installation
+In this context, _metadata_ means information about executions (runs), models, 
+datasets, and other artifacts. _Artifacts_ are the files and objects that form 
+the inputs and outputs of the components in your ML workflow.
 
-The Metadata component is installed by default for Kubeflow versions >= 0.6.1.
+{{% alert title="Alpha version" color="warning" %}}
+This is an <b>alpha</b> release of the Metadata API. The next version of Kubeflow
+will introduce breaking changes. The development team is interested in any
```

Comment:
I would change `will` to `may`, because I think it is possible to keep current REST api the same and I don't want to scare people away.

Reply:
OK, thanks - I'll let this PR go through with your LGTM and send a follow-up PR to change "will" to "may".